Packs/Base/Scripts/DBotPreprocessTextData/DBotPreprocessTextData.py

```diff
@@ -378,9 +408,16 @@ def main():
     remove_html_tags = demisto.args()['cleanHTML'] == 'true'
     whitelist_fields = demisto.args().get('whitelistFields').split(",") if demisto.args().get(
         'whitelistFields') else None
+    language = demisto.args().get('language', 'English')
     # if input is a snigle string (from DbotPredictPhishingWords):
     if input_type == 'string':
-        res = pre_process_single_text(raw_text=demisto.args().get('input'),
+        input_str = demisto.args().get('input')
+        input_str = clean_text_of_single_text(input_str, remove_html_tags)
```

Comment:
I don't think we should clean the text in the user didn't specify any input for that, is will effect the script preformance.
Maybe we should add  boolean argument, to set if we want to filter or not
WDYT?

Reply:
the `remove_html_tags` is an argument taken from the script arguments, and unless set to `true` it will only clean line breaks. Got your question correctly?

Reply:
I talked about the lang detectio, not the html cleaning